packages/liveblocks-react-ui/src/components/AiChat.tsx

```diff
@@ -205,7 +205,7 @@ export const AiChat = forwardRef<HTMLDivElement, AiChatProps>(
           ) : (
             <>
               <AutoScrollToBottomHandler
-                lastSendMessageId={lastSendMessageId}
+                lastSendMessageId={lastSentMessageId}
```

Comment:
Thanks for renaming this. Did you forget to also rename the `lastSendMessageId={}` prop here?

Reply:
The typo is now fixed everywhere 👍

Yes I also prefer the "scroll last user message to the viewport's top" behavior like a few chat apps now do, I renamed the helper to `AutoScrollHandler` but it's internal so it doesn't really matter for now. I think we need a whole separate PR on scroll tactics (maybe post-launch?):
- Should it stick to the bottom? We internally don't want that but some users might?
- On submit, when scrolling to the last user message, should it put it at the viewport's top or bottom?
- Something else?'[]]